drivers/TimerEvent.h

```diff
@@ -44,9 +44,12 @@ class TimerEvent {
     // The handler called to service the timer event of the derived class
     virtual void handler() = 0;
 
-    // insert in to linked list
+    // insert relative timestamp in to linked list
```

Comment:
I guess this was meant to be "relative to the 32-bit cycle" rather than "relative current time". But later contributors assumed the later, and it's only addressed by #14005.